### PR TITLE
[jsk_fetch_startup] add fetch etc files

### PR DESCRIPTION
**Pull request by sktometometo**
- add upstart scripts and launch files which are usually in /etc/init/ and /etc/ros/indigo.
- add a script to deploy the upstart scripts
- modify the upstart scripts so that they will use jsk_fetch_startup version of launch files instead of robot.launch in /etc/ros/indigo

### CODE REVIEW

File: jsk_fetch_robot/jsk_fetch_startup/launch/fetch.launch [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  <arg unless="$(arg debug)" name="launch_prefix" value=""/>
+  <arg if="$(arg debug)" name="launch_prefix" value="gdb -x $(find fetch_bringup)/launch/gdb_settings.gdb --ex run --args"/>
+
+  <!-- calibration: s-kitagawa (2019/12/11) -->
+  <!-- Calibration -->
+  <param name="calibration_date" value="2019-12-11 03:27:27"/>
+  <param name="base_controller/track_width" value="0.37476"/>
+  <param name="head_camera/driver/z_offset_mm" value="4"/>
+  <param name="head_camera/driver/z_scaling" value="1.0202112"/>
+  <arg name="rgb_camera_info_url" default="file:///etc/ros/indigo/depth_2019_12_11_03_27_26.yaml"/>
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch_startup/launch/fetch.launch]: Oh, I totally forgot it but camera calibration file should be also a simulink

**Comment by knorth55** [comment]
@sktometometo I made a PR to update latest upstart scripts and fetch.launch.
please merge this PR and remove WIP label. https://github.com/sktometometo/jsk_robot/pull/1

**Comment by knorth55** [comment]
@sktometometo @708yamaguchi we can also replace `install_upstart.sh` script to this `deploy_upstart_script.sh`.
what do you think?

**Comment by sktometometo** [comment]
@knorth55 which install_upstart.sh do you mean?

**Comment by knorth55** [comment]
@sktometometo this one https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_fetch_robot/jsk_fetch_startup/config/install_upstart.sh

**Comment by 708yamaguchi** [comment]
@knorth55 

> we can also replace install_upstart.sh script to this deploy_upstart_script.sh. (or rename deploy_upstart_script.sh to install_upstart.sh)
> what do you think?

I think it's good idea to use `deploy_script.sh` because it is understandable than `install_upstart.sh`.

**Comment by knorth55** [comment]
@sktometometo please merge this pr https://github.com/sktometometo/jsk_robot/pull/2

**Comment by knorth55** [comment]
i rename from `deploy_upstart_scripts.sh` to `install_upstart.sh`

**Review by k-okada — COMMENTED** [review]
@sktometometo sorry, I merged #1328, so please fix conflicts

File: jsk_fetch_robot/jsk_fetch_startup/scripts/network_monitor.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 ## NETWORK_MONITOR_ENABLED : Set to 'false' to disable script
 
 desc = ("Monitors network status and restart network manager when needed. "
-        "Script is expected to be run as an upstart job. If run manually, "
+        "Script is expected to be run as an upstart job. If run manually, ">>>>>>> master
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch_startup/scripts/network_monitor.py:17]: !!!!

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: jsk_fetch_robot/jsk_fetch_startup/scripts/network_monitor.py (hunk 2 above)
Comment (sktometometo) [reply jsk_fetch_robot/jsk_fetch_startup/scripts/network_monitor.py:17]: sorry for mistake.

**Review by knorth55 — APPROVED** [review]

**Comment by k-okada** [comment]
@knorth55 if possible, please approve PR, after all CI is passed, so that I can push merge button, when I checked notification.

**Comment by knorth55** [comment]
@k-okada OK. I will.

**Pull request merged**